app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -111,19 +111,27 @@ def update_classification
       form526_submission_id: id
     }
 
+    Rails.logger.info(
+      "classifying 526Submission... id: #{id}, saved_claim_id: #{saved_claim_id}"
+    )
     classification = classify_by_diagnostic_code(params)
-    update_form_with_classification(classification['classification_code']) if classification.present?
+    update_form_with_classification(classification)
   end
 
-  # check claims
   def classify_by_diagnostic_code(params)
     vro_client = VirtualRegionalOffice::Client.new
     response = vro_client.classify_contention_by_diagnostic_code(params)
     response.body
   end
 
-  def update_form_with_classification(_classification_code)
-    # TODO: update form[FORM_526] to include the classification code
+  def update_form_with_classification(classification)
+    classification_code = classification['classification_code'] if classification.present?
+    form[Form526Submission::FORM_526]['form526']['disabilities'].each do |disability|
```

Comment:
Are you changing ALL the contention classification code, for ALL disabilities? Is this only being used one claims with 1 single disability? This is also probably fine, if you know for sure this will only get triggered when there is only 1 disability (strange to be using a loop then though), but it would not be good to overwrite all the codes, if there are multiple, with the code from 1 of the disabilities. 

Again, I dont think this will error or be an issue NOW, since you are only doing this when there is 1 disability, but could be trouble in the future!

Reply:
[HERE](https://github.com/department-of-veterans-affairs/vets-api/blob/bugfix-cc-endpoint/app/models/concerns/form526_rapid_ready_for_decision_concern.rb#L102) are the guards for `update_classification` to reduce the scope of claims. I think this will require more significant refactor when we move to handling submissions with multiple contentions in the next couple months

Reply:
> Do you want the contention code given to be overwritten with nil if the service returns nil? (what it looks like it is currently doing) or is it supposed to keep the code it had if the service does not return a code?

Talked w/ Kyle on this one via slack. Possible api response of null, test should now cover that